LondonBritishLibrary/orient/BLorient12996.xml

```diff
@@ -0,0 +1,289 @@
+                                <msItem xml:id="p1_i1.11">
+                                    <locus from="21r"/>
+                                    <title ref="STU0009AmharicGram#Participle">De Participio sive Supino in Rus.</title>
+                                    <note>On participles (sic) of the pattern 'supino' (i.e. 'supine') in Rus. (i.e. in Russian?)</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.12">
+                                    <locus from="22v"/>
+                                    <title ref="STU0009AmharicGram#ParticipleAgentis">De Participijs Agentibus et Patientibus</title>
+                                </msItem>
+                                <msItem xml:id="p1_i1.13">
+                                    <locus from="26r"/>
+                                    <title ref="STU0009AmharicGram#DeclensionParticiples">Declinatio supradictorum duorum Participii</title>
+                                </msItem>
+                                <msItem xml:id="p1_i1.14">
+                                    <locus from="28v"/>
+                                    <title ref="STU0009AmharicGram#VerbalNoun">De Abstracto seu Nomine Verbali</title>
+                                    <note>On the verbal noun e.g. the gerund.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.15">
+                                    <locus from="32r"/>
+                                    <title ref="STU0009AmharicGram#TablePronounAffixes">Tabula Pronominum Verbis affixorum</title>
+                                    <note>Table showing the pronominal affixes as they appear on the verb.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.16">
+                                    <locus from="32v"/>
+                                    <title ref="STU0009AmharicGram#TablePresent">Secunda Tabula pro Tempore Praesenti</title>
+                                    <note>Second table shows the verbal forms of the present tense.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.17">
+                                    <locus from="34r"/>
+                                    <title ref="STU0009AmharicGram#Negative">Conjugazio Verbi Negativi, Modo Arabico</title>
+                                    <note>Conjugation of negated verbs in the Arabic mode (sic).</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.18">
+                                    <locus from="35v"/>
+                                    <title ref="STU0009AmharicGram#Dicere">Conjug. Verbi Affirmativi Dicere in Amahara</title>
+                                </msItem>
+                                <msItem xml:id="p1_i1.19">
+                                    <locus from="0000"/>
```

Comment:
Why zeros?

Reply:
I forgot to insert the leave. The correct number is 37v.